pygmt/tests/test_meca.py

```diff
@@ -54,7 +118,7 @@ def test_meca_spec_dict_list():
 
 
 @pytest.mark.mpl_image_compare
-def test_meca_spec_dataframe():
+def test_meca_spec_two_dataframe():
```

Comment:
Why is this being renamed? As far as I can tell there is just a single DataFrame in this function.

Reply:
I was planning to have two groups of tests:

1. In tests `test_meca_spec_*`, a single focal mechanism in different data types (dict, DataFrame, 1darray and plaintext file) is passed to the `spec` parameter
2. In tests `test_meca_spec_two_*`, two focal mechanisms in different data types (dict with list as keys, DataFrame, 2D list, 2D array and plaintext file) are passed to the `spec` parameter.

That's why I added the `_two` suffix to the tests, but I agree that's not a good name for these tests. Do you have better ideas?

Reply:
That makes sense. I think it will be hard to come up with a descriptive test name without making it too wordy. I think it might just be easier to update the docstring (I made my suggestion below) to make it clear that it's passing two focal mechanisms.